libs/core/src/lib/package.ts

```diff
@@ -286,6 +287,24 @@ export class Package {
     return writePkg(this.manifestLocation, this[PKG] as any).then(() => this);
   }
 
+  /**
+   * Sync dist manifest version
+   */
+  syncDistVersion() {
```

Comment:
As mentioned in my review, this function can be converted to use `async/await` instead of `Promise.resolve` and `.then` chaining.